javascript/ql/src/semmle/javascript/Functions.qll

```diff
@@ -158,6 +158,51 @@ class Function extends @function, Parameterized, TypeParameterized, StmtContaine
     result = getAReturnStmt().getExpr()
   }
 
+  /** 
+   * Gets a return from a function which has undefined value (i.e. implicit returns
+   * and returns w/o expressions).
+   */
+   
+  /* Functions can have undefined returns in a few different ways:
```

Comment:
You can just join this comment with the previous one. Generally, the first paragraph of a doc comment gives an overview (as the one above does), while the rest of the comment provides technical details.